src/OrbitGl/CallstackThreadBar.cpp

```diff
@@ -116,7 +118,8 @@ void CallstackThreadBar::DoUpdatePrimitives(PrimitiveAssembler& primitive_assemb
     auto action_on_callstack_events = [&](const CallstackEvent& event) {
       const uint64_t time = event.timestamp_ns();
       ORBIT_CHECK(time >= min_tick && time <= max_tick);
-      Vec2 pos(timeline_info_->GetWorldFromTick(time), GetPos()[1]);
+      Vec2 pos(GetPixelNumber(event.timestamp_ns(), resolution_in_pixels, min_tick, max_tick),
```

Comment:
Irrelevant to this PR, but IMO we either const all of the constable local variables, or none.

Reply:
I agree with what you said although I might prefer in some cases to have some const statements.

I will const all the lines I changed but I won't add const in previous existent variables.